agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.configuration;
+
+import io.opentelemetry.javaagent.bootstrap.servlet.ExperimentalSnippetHolder;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+// preview Configuration
+public class SnippetConfiguration {
+  private static String snippet = readSnippet();
+
+  public static String readSnippet(){
```

Comment:
can you add a test for readSnippet?

Reply:
or better a smoke test for snippet injection?

Reply:
the smoke test for snippet injection is written, but have a wired bug, would work on that next week.